test/extended/prometheus/prometheus.go

```diff
@@ -195,6 +195,17 @@ var _ = g.Describe("[Feature:Prometheus][Conformance] Prometheus", func() {
 			}
 			runQueries(tests, oc, ns, execPod.Name, url, bearerToken)
 		})
+		g.It("shouldn't experience cardinality explosion", func() {
+			oc.SetupProject()
+			ns := oc.Namespace()
+			execPod := exutil.CreateCentosExecPodOrFail(oc.AdminKubeClient(), ns, "execpod", nil)
+			defer func() { oc.AdminKubeClient().CoreV1().Pods(ns).Delete(execPod.Name, metav1.NewDeleteOptions(1)) }()
+
+			tests := map[string]bool{
+				`rate(scrape_series_added[10m]) > 60`: false, // Threshold experimentally chosen
```

Comment:
Is 10 minutes good time range for this query, should we do longer?

Reply:
Hmm, `scrape_series_added` is a gauge so `rate` doesn't work here. `deriv` or `avg_over_time` are probably best suited?

Reply:
We can extend it. I am experimenting with different ranges :)